docs-src/modules/getting-started/pages/index.adoc

```diff
@@ -1,14 +1,91 @@
 = Getting Started
```

Comment:
Feels like it would be good to have an intro paragraph between the "getting started" and the first section (reasoning about veents with timelines).